dockstore-integration-testing/src/test/java/io/dockstore/client/cli/LimitedCRUDClientIT.java

```diff
@@ -254,13 +273,25 @@ public void testUploadZipHonorsVersionLimit() {
 
     }
 
-    private List<SourceFile> generateSourceFiles() throws IOException {
+    private List<SourceFile> generateSourceFiles(String lang) throws IOException {
+        String resourceFilePath;
+        String dockstorePath;
+        SourceFile.TypeEnum type;
+        if (lang.equals(CWL.getLowerShortName())) {
+            resourceFilePath = "tar-param.cwl";
+            dockstorePath = "/Dockstore.cwl";
+            type = SourceFile.TypeEnum.DOCKSTORE_CWL;
+        } else {
+            resourceFilePath = "hello.wdl";
+            dockstorePath = "/Dockstore.wdl";
+            type = SourceFile.TypeEnum.DOCKSTORE_WDL;
+        }
```

Comment:
should probably fast fail if it is an unexpected language (should not assume WDL)